### ISSUE TITLE
Spike of crop ratios options for media browser form

### ISSUE THREAD

**Issue by cmaeng** [department-of-veterans-affairs/va.gov-cms#3969]
## User Story or Problem Statement
CLP has requested that the crop ratio for the hero panel be restricted to 2:1. Need a spike to audit existing content types to look for fields that use images, solve for this at a macro level, and then see if this solves the CLP specific use case.

## Acceptance Criteria
- [ ] Research if the upgrade from 8.8 to 8.9 addresses this in an easy way
- [ ] Explore the solution of:
  1. Look in drupal for places that aren't using 2:1 image aspect ratio
  2. Update front end graphql and templates to use 2:1 ratio
  3. Limit drupal fields to only show 2:1 ratio option (or maybe none at all, if this is the only option)

**Comment by ethanteague** [comment]
I know this is a spike with acs to document the different cropping styles that should be used in various places, but the end goal is to figure out how to not have the same multiple options appearing everywhere the imagecrop widget is in play. I've been puzzling / researching this for a while. The crop api doesn't offer any ootb the solutions for this, and neither does the ImageCropWidget. There isn't any path to improve this documented for D9, either. So, I think the only way to limit crop options on a content type (or paragraph type) basis is to rework the content model, and apply a unique cropping style on a per field basis (the challenge we are having now is that the image library browser uses the same image field everywhere it appears, and doesn't limit the cropping styles). This would mean we would need to create a new image field for each crop style, and then work out how to display it in an entity browser, and then plug in the new field and browser on each content type that needed said field type.
Tldr: big lift

Per convo with @kevwalsh in slack, we are gonna punt on this one (at least in sprint 25).

**Comment by EWashb** [comment]
Per PO in sprint 63 planning, before we merge this we need to determine a research plan including metrics for success of this work. I will create an issue for research planning
